src/experimental/slam_toolbox_lifelong_node.cpp

```diff
@@ -42,8 +42,6 @@ int main(int argc, char ** argv)
 
   rclcpp::NodeOptions options;
   auto lifelong_node = std::make_shared<slam_toolbox::LifelongSlamToolbox>(options);
-  lifelong_node->configure();
```

Comment:
Where are these called now?

Reply:
It is called in `SlamToolbox::on_configure`

https://github.com/SteveMacenski/slam_toolbox/blob/78af8485ebfd7d2b95e50188a341b86b4069e644/src/slam_toolbox_common.cpp#L51-L63

Specifically, when the lifecycle is configured , ...

`LifelongSlamToolbox::on_configured` -> `SlamToolbox::on_configured` -> `configure`

https://github.com/SteveMacenski/slam_toolbox/blob/78af8485ebfd7d2b95e50188a341b86b4069e644/src/experimental/slam_toolbox_lifelong.cpp#L70-L78